mqttwarn.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Kindly undo that change for the time being. That path may be correct for your system but it isn't in general.

Reply:
Thought this was pretty general for Linux, but sure, will change it

Reply:
It may well be, but there's not just Linux in the world. :-)